docs/source/1.0/spec/aws/aws-auth.rst

```diff
@@ -148,6 +148,33 @@ literal string ``UNSIGNED-PAYLOAD`` is used when constructing a
 .. smithy-trait:: aws.auth#cognitoUserPools
 .. _aws.auth#cognitoUserPools-trait:
 
+------------------------
+``presignable`` trait
+------------------------
```

Comment:
You'll need to add a `smithy-trait` directive here and make sure that the cognito user pools directive is directly above its trait. Also the title here should be fully qualified, e.g. `aws.auth#presignable` (to be fair, we're inconsistent on this currently)

Reply:
Ah, I didn't notice the significance of these directives.  Fixed, thanks.